csrc/mmdeploy/net/torchscript/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+project(mmdeploy_torch_net)
+
+find_package(Torch REQUIRED)
+find_package(TorchVision)
+
+mmdeploy_add_module(${PROJECT_NAME} torch_net.cpp)
+
+target_link_libraries(${PROJECT_NAME} PRIVATE
+        ${TORCH_LIBRARIES})
+
+target_link_directories(${PROJECT_NAME} INTERFACE
+        $<BUILD_INTERFACE:${Torch_DIR}/../../../lib>)
+
+target_link_libraries(${PROJECT_NAME} PRIVATE
+        mmdeploy_torchscript_ops_obj)
+
+if ("cuda" IN_LIST MMDEPLOY_TARGET_DEVICES)
```

Comment:
`MMDEPLOY_USE_CUDA` could be placed outside `torchscript/CMakeLists.txt` since it might be used by other backend or transforms.